pyccel/codegen/printing/luacode.py

```diff
@@ -357,7 +358,7 @@ def _print_Piecewise(self, expr):
         for i, (e, c) in enumerate(expr.args):
             if i == 0:
                 lines.append("if (%s) {" % self._print(c))
-            elif i == len(expr.args) - 1 and c == True:
+            elif i == len(expr.args) - 1 and c is True:
```

Comment:
Similarly to above pylint suggests using `elif i == len(expr.args) - 1 and c:` here. Do you find this more readable?

Reply:
Fine with me!